job-dsls/jobs/new_downstream_production.groovy

```diff
@@ -0,0 +1,171 @@
+/**
+ * Creates full downstream pullrequest (PR) jobs for kiegroup GitHub org. units.
+ * These jobs execute the full downstream build for a specific PR to make sure the changes do not break the downstream repos.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def final DEFAULTS = [
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        branch                 : Constants.BRANCH,
+        timeoutMins            : 650,
+        label                  : "kie-rhel7 && kie-mem24g",
+        ghAuthTokenId          : "kie-ci-token",
+        ghJenkinsfilePwd       : "kie-ci",
+        artifactsToArchive     : [
+                ",**/target/screenshots/**",
+                "**/target/kie-server-*ee7.war",
+                "**/target/kie-server-*webc.war",
+                "**/target/jbpm-server*dist*.zip",
+                "**/target/business-monitoring-webapp.war",
+                "**/target/business-central*wildfly*.war",
+                "**/target/business-central*eap*.war"
+        ]
+]
+// override default config for specific repos (if needed)
+def final REPO_CONFIGS = [
+        "lienzo-core"               : [],
+        "lienzo-tests"              : [],
+        "droolsjbpm-build-bootstrap": [],
+        "kie-soup"                  : [],
+        "appformer"                 : [],
+        "droolsjbpm-knowledge"      : [],
+        "drools"                    : [],
+        "optaplanner"               : [],
+        "jbpm"                      : [],
+        "kie-jpmml-integration"     : [],
+        "droolsjbpm-integration"    : [],
+        "openshift-drools-hacep"    : [],
+        //"droolsjbpm-tools"          : [], // no other repo depends on droolsjbpm-tools
+        "kie-wb-playground"         : [],
+        "kie-uberfire-extensions"   : [],
+        "kie-wb-common"             : [
+                label               : "kie-rhel7&&kie-mem24g&&gui-testing"
```

Comment:
this is not needed since `gui-testing` is not needed by kie-wb-common and then becomes the default value

Reply:
taken out